include/zephyr/net/wifi.h

```diff
@@ -27,6 +27,9 @@
 #define WIFI_LISTEN_INTERVAL_MIN 0
 #define WIFI_LISTEN_INTERVAL_MAX 65535
 
+/** AP mode maximum number of STA entries */
+#define WIFI_AP_MODE_MAX_NUM_STA 4
```

Comment:
This looks unused and enforcing a random limit, so, may be remove?

Reply:
Added the check for this value

Reply:
As discussed, we have decided to enforce 802.11 protocol limits, 4 is just the default. And we already have a Kconfig: `WIFI_MGMT_AP_MAX_NUM_STA`, so, please use that.